app/views/series/index.html.erb

```diff
@@ -1,35 +1,37 @@
 <div id="notice"><%= notice %></div>
 
-<h1>Series Listings</h1>
+<html lang="en-US">
```

Comment:
Is this addition needed?  I would have thought that your fix #188 would have been sufficient to address this.  I believe this specific change would lead to an `<html>` element inside another `<html>` element, which [is not valid](https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/html).